ui/web_gui.py

```diff
@@ -36,7 +42,6 @@ def create_fastapi_app(self):
                                allow_headers=['*'])
 
         app.mount("/panel/static", StaticFiles(directory="/srv/frontend/static"), name="panel")
-        app.mount("/static", StaticFiles(directory="/repo/data/web_gui/static"), name="static")
```

Comment:
Oh was this used for putting gstreamer files, etc. ? I think I now understand what you did, maybe it is better to change the name if it has to contain stream related files (playlist, video chunk, etc.)

Reply:
Read several lines below and got it. Yeah, I think web_gui is not making sense as a name anymore, it's confusing.

Reply:
Right. I should make a more thorough search for web GUI mentions

Reply:
So Web_GUI now is the main class which serves the frontend files. It is decoupled from the video processing part. I replaced 'web_gui' with 'processor' in the path that serves the processed video stream.